empack_config.yaml

```
@@ -0,0 +1,48 @@
+packages:
+  python-dateutil: 
+    include_patterns:
+      - pattern: '*.so'
+      - pattern: '*.py'
+      - regex: '*dateutil-zoneinfo.tar.gz'
+   exclude_patterns:
+      - pattern: '**/tests/**/*.py'
+      - pattern: '**/tests/**/*.so'
+
+  matplotlib:
+    include_patterns:
+      - pattern: '*.so'
+      - pattern: '*.py'
+      - regex: '*dateutil-zoneinfo.tar.gz'
```

It seems this is not the latest version that fixed Matplotlib.

Can you also add `fontlist.json`? In combination with https://github.com/emscripten-forge/recipes/pull/142 we should get a decent import time improvement for matplotlib.

```suggestion
      - regex: '*dateutil-zoneinfo.tar.gz'
      - pattern: '**/matplotlib/mpl-data/**'
      - patter: '**/matplotlib/fontlist.json'
```